packages/utils/src/script.ts

```diff
@@ -11,6 +11,24 @@ import {
 import { decodeInt, encodeInt } from './data';
 import OptimisationsEquivFile from './cashproof-optimisations';
 
+// TODO: Replace this when these opcodes are in Libauth
+export enum IntrospectionOp {
```

Comment:
I couldn't find why you introduced a separate class/namespacing from `Op`. Is it some kind of missing auto-mapping with libauth? This is just my curiosity. Not suggesting any changes.

Reply:
Yeah so `Op` uses Libauth's named opcodes, which doesn't include these new ops yet. I could add the to the `Op` namespace but that would require me to struggle with convincing TypeScript that everything is ok, so this was the easier solution. As the comment states, I'm going to remove this any way once Libauth is updated for May 2022.